infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/EventRemoteInterface.kt

```diff
@@ -37,6 +38,7 @@ internal interface EventRemoteInterface : SimRemoteInterface {
         @Query("lastEventId") lastEventId: String?
     ): ResponseBody
 
+    @Headers("Content-Encoding: gzip")
```

Comment:
FYI we have not added support for content encoding on this endpoint on our side.
But it makes sense, I'll add it early next week.